code/povelikin-rostislav/src/test/java/ru/unn/agile/redblacktree/RedBlackTreeTest.java

```diff
@@ -0,0 +1,230 @@
+package ru.unn.agile.redblacktree;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.*;
+
+import static junit.framework.TestCase.*;
+
+public class RedBlackTreeTest {
```

Comment:
I didn't find test that tree will be created balanced, not like a list. I think it can be done with tree height tests.

Reply:
It is out of the scope, I think, because TDD is used to prototype API/design and there is no any sense (as I see) to expose the internal implementation details like tree height.

Even if tree is implemented as an ordinary list - it will provide an API user wants. It want be optimal, but that would be a problem of performance tests, I suppose.